lib/unhangout-routes.js

```diff
@@ -43,6 +46,10 @@ module.exports = {
             res.render('about.ejs', {user: req.user});
         });
 
+        app.get("/contact_info/", function(req, res) {
+            res.render('contact_info.ejs', {user: req.user});
+        });
```

Comment:
Since contact info is per-event, this route needs to include an event ID to determine which event's contact info is being changed.